ONSdigital/florence
# Release 1.13.8

### What

* Test files and missed JSX files formatter
* Webpack prettier plugin removed - inaccurate
* Prettier node module added
* Prettier node scripts added
* Linter rule check node script added
* Prettier added to CI checks
* Prettier formatting rules moved into an RC file
* Radio buttons on Florence should now appear correctly on Windows
* ESLint updated
* ESLint now extends prettier formatter rules

### How to review

1. Try make a bad formatting change to the code base
2. Build files
3. Files should be formatted automatically
4. Turn off formatting by removing prettier scrip call in the package.json file 
5. Repeat steps 1&2
6. Create a pull request with the bad formatting
7. CI concourse should fail the build for not being formatted correctly
8. Fix the formatting and push changes
9. CI concourse should approve the changes and pass the PR 
10. Delete the PR and clean up after testing

### Who can review

Anyone except me
